Library/Formula/ratfor.rb

```diff
@@ -0,0 +1,15 @@
+class Ratfor < Formula
+  version "1.02"
+  desc "Rational Fortran"
+  homepage "http://www.dgate.org/ratfor/"
+  url "http://www.dgate.org/ratfor/ratfor.tar.gz"
+  sha256 "daf2971df48c3b3908c6788c4c6b3cdfb4eaad21ec819eee70a060956736ea1c"
+
+  depends_on "gcc"
```

Comment:
No, for use of the Fortran code generated. I guess that should be noted rather than a requirement for installation.